libpsautohint/autohintexe.c

```diff
@@ -401,25 +401,8 @@ main(int argc, char* argv[])
                 openReportFile(bezName, fileSuffix);
             }
 
-            result = AutoColorString(bezdata, fontinfo, output, &outputsize,
+            result = AutoColorString(bezdata, fontinfo, &output, &outputsize,
                                      allowEdit, allowHintSub, roundCoords, debug);
-            if (result == AC_DestBuffOfloError) {
```

Comment:
I'm not sure if it's OK for this to fall through to the return or not or if you should still check result at this point.